data/human/weapons.txt

```diff
@@ -11,6 +11,194 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Tethys Cannon"
+	category "Turrets"
+	cost 1067000
+	mass 128
+	thumbnail "outfit/rnd_gun2_turret"
+	"outfit space" -110
+	"weapon capacity" -97
+	"drag" 7
```

Comment:
I don't think we should be adding drag to turrets, none of the other turrets or weapons do. Even ones almost as large as this one.